lib/vscode/src/vs/platform/environment/common/environmentService.ts

```diff
@@ -15,6 +15,24 @@ import { URI } from 'vs/base/common/uri';
 import { ExtensionKind } from 'vs/platform/extensions/common/extensions';
 import { env } from 'vs/base/common/process';
 
+// NOTE@coder: add parsePathArg
+function parsePathArg(arg: string | undefined, process: NodeJS.Process): string | undefined {
+	if (!arg) {
+		return undefined;
+	}
+
+	// Determine if the arg is relative or absolute, if relative use the original CWD
+	// (VSCODE_CWD), not the potentially overridden one (process.cwd()).
+	const resolved = resolve(arg);
+
+	if (normalize(arg) === resolved) {
+		return resolved;
+	}
+
+	return resolve(process.env['VSCODE_CWD'] || process.cwd(), arg);
+}
+
+
```

Comment:
If VS Code no longer uses this we can get rid of it as well. I believe it was used because `extensions-dir` was using it so we can match what they do there. Looks like they use `resolve` now.

Reply:
Ah yup I followed `resolve` down and they handle `VSCODE_CWD` further down the line so we're good to just use `resolve` directly.

Reply:
done!